snuba/admin/static/clickhouse_migrations/index.tsx

```diff
@@ -61,6 +60,28 @@ function ClickhouseMigrations(props: { api: Client }) {
     console.log("executing !", action);
   }
 
+  function executeDryRun(action: Action) {
+      let req = {
+        action: action,
+        migration_id: migrationId,
+        group: migrationGroup?.group,
+        dry_run: true
+      }
+      console.log("executing dry run !", migrationId, action);
+      console.assert(req.dry_run, "dry_run must be set")
+      props.api
+      .runMigration(req as RunMigrationRequest)
+      .then((res) => {
+        console.log(res)
+        setSQLText(() => res.stdout);
+      })
+      .catch((err) => {
+        console.log(err)
+        setSQLText(() => JSON.stringify(err));
```

Comment:
we should be able to do `err.error` should we not?

Reply:
This is a more generic error type and can include both client and server side response errors. For server side we can do `err.error`, client side we cannot.